net-ftp/uftpd/uftpd-2.5.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="The no nonsense TFTP/FTP server"
+HOMEPAGE="https://github.com/troglobit/uftpd"
+SRC_URI="https://github.com/troglobit/${PN}/releases/download/v${PV}/${P}.tar.xz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+		!net-misc/uftp
+		!net-ftp/atftp"
+
+DEPEND="
+	dev-libs/libite
+	dev-libs/libuev"
+
+RDEPEND="${DEPEND}"
```

Comment:
Ok, this makes no sense. You've setting `RDEPEND` first, then overwriting it here.